src/deprecated.jl

```diff
@@ -18,5 +18,5 @@ import Base: keys, values, insert!
 
 @deprecate sub(dt::AbstractDataTable, rows) view(dt, rows)
 
-@deprecate stackdf stackdt
-@deprecate meltdf meltdt
+@deprecate stackdf stack
```

Comment:
I'd revert it for now, as it's unrelated. We can discuss steering people toward `stack` separately.